src/odemis/driver/symphotime.py

```diff
@@ -856,7 +856,7 @@ def _actOnMessage(self,decoded_msg):
                 logging.warning("Measurement data frame received while not in measuring state. ")
 
             # convert the OptionalDataRecord type to a regular python dictionary for logging
-            data = {r.name : r.data for r in decoded_msg.odOptional.values()}
+            data = {r.name.decode('latin1') : r.data for r in decoded_msg.odOptional.values()}
```

Comment:
Instead of decoding here, I'd suggest to decode in `from_bytes()`. Maybe use "utf-8" with replace error handler, to be symmetrical with the encoding used in `to_bytes()`